src/pages/HomePage/index.js

```diff
@@ -1,29 +1,59 @@
 import React, { Component } from "react";
+import windowSize from "react-window-size";
+
 import articles from "../../fixtures/articles.js";
 import persons from "../../fixtures/persons.js";
 import OurStoriesArticles from "../../components/OurStoriesArticles";
 import ButtonsSocial from "../../components/ButtonsSocial";
 import ButtonList from "../../components/ButtonList";
 import "./style.css";
 import TriangleRight from "../../icons/TriangleRight";
-import CustomerGroups from "../../components/CustomerGroups";
+import SliderWrapper from "../../components/SliderWrapper";
 
 class HomePage extends Component {
   state = {
     defaultPerson: {},
     numberPhoto: "01",
-    currentIndex: 0
+    currentIndex: 0,
+    sliderWrappLength: 0,
+    maxItemLength: 0,
+    listLength: null,
+    bigMonitor: 8,
+    mediumMonitor: 6,
+    smallMonitor: 4,
+    tinyMonitor: 2,
```

Comment:
if you use it like constant don't define it in state. Just create such variables

```
const LARGE_MONITOR = 8
```
but better if you named this variable something like this
`const SLIDER_IMG_LENGTH_LARGE = 8`